docs/sources/tutorials/get-started.md

```diff
@@ -14,21 +14,40 @@ You don't need to know about the {{< param "PRODUCT_NAME" >}} [configuration syn
 
 ## Prerequisites
 
-The tutorials require a Linux or Unix environment with Docker installed.
+This first tutorial requires a Linux, Unix, or Mac environment with Docker installed.
 The examples run on a single host so that you can run them on your laptop or in a Virtual Machine.
 You are encouraged to try the examples using a `config.alloy` file and experiment with the examples yourself.
 
```

Comment:
I propose we replace this entire section with the following sentence. The other info included in the original section has been placed in other parts of the document where it's more relevant. 

This tutorial requires a Linux, Unix, or Mac environment with Docker installed.